### DISCUSSION TITLE
How do I un-archive a habit?

### DISCUSSION THREAD

**Discussion by pseudacris** [iSoron/uhabits#1667] [Help & FAQ]
I can't figure out how to un-archive a habit. I think it would be good to add it to the FAQ.

**Comment by dimo414** [comment]
If you long-press the habit in the list view an "Unarchive" option appears in the three-dot menu.

**Reply by ajgower455** [reply]
How can you long press the habit when it disappears once archived?!

**Reply by dimo414** [reply]
There's a "Hide archived" filter that is checked by default.

**Comment by ajgower455** [comment]
Archived habits do not appear to be visible anywhere. So a) how can you unarchive them and b) how is archive distinct from delete if you can't access archived habits?

**Comment by pseudacris** [comment]
You can access them. I was confused too, it's not in the FAQ last I
checked.

At the top there's a filter button that looks like an arrowhead pointing
down made of lines. Click that, uncheck the box that says "Hide archived"
and your archived habits will reappear and you can long press them.
